rotkehlchen/db/history_events.py

```diff
@@ -81,6 +81,7 @@ class DBHistoryEvents:
 
     def __init__(self, database: 'DBHandler') -> None:
         self.db = database
+        self.free_history_events_limit = FREE_HISTORY_EVENTS_LIMIT
```

Comment:
This allowed me to mock it in the test, and also we may need it as a variable when allowing different tiers of premium. But, lmk if we should still keep it as constant, it's tricky because the test is interpreted after the constant is imported so the mocked value is not used in the test case.

Reply:
Yes, you shouldn't have code just for tests. It is possible to mock a global variable too, we do it in other places and for the tier premium is an adjustment that we can do later. Is not a significant change